modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -63,7 +63,26 @@ spec:
 . *Deploy the Operator.* Use the files that were generated into the `deploy/`
 directory by the Operator SDK when you built the Memcached Operator.
 
-.. Apply the Operator's CSV manifest to the specified namespace in the cluster:
+.. Edit the generated CSV manifest file by adding `description` and `displayName` fields for each custom resource definition (CRD) `kind` in the `spec.customresourcedefinitions.owned` section:
++
+.deploy/olm-catalog/memcached-operator/0.0.1/memcached-operator.v0.0.1.clusterserviceversion.yaml file
+[source,yaml]
+----
+...
```

Comment:
Not sure why the `...` is purply in the preview rendering. Could be old CSS in 4.3? Could also just be the preview build. 

I found other oddness in another 4.3 assembly: https://docs.openshift.com/container-platform/4.3/nodes/scheduling/nodes-scheduler-node-selectors.html#nodes-scheduler-node-selectors-pod_nodes-scheduler-node-selectors
If you look at the different samples in ^^ step 2, the `...` is underlined but `....` is fine. Maybe add a period to yours? Or just let it ride cuz 4.3.

Reply:
I believe it's always been/is still like this for source YAML highlighting for `...`, which is very unfortunate. I'll double check tho.

Reply:
@mikemckiernan found an alternate, I think.

Reply:
Went ahead and just removed the `[source,yaml]` here since most of the rest of the assembly isn't using it anyway back in 4.3. I had put it in because the OperatorGroup example right above did have it; but I've removed it there also for consistency within the doc.